.zenodo.json

```diff
@@ -109,6 +114,11 @@
     {
       "name": "Hart, Gabe"
     },
+    {
```

Comment:
This is now a duplicate of line 80.

Reply:
I suspect the other additions are duplicates, too (except Lee and Mihail).

Reply:
Duplicate removed.

Reply:
I checked for duplicates on the others.